src/components/m-accordion-item/js/accordion-item.js

```diff
@@ -0,0 +1,163 @@
+import UiEvents from '../../../js/ui-events';
+import on from '../../../js/on';
+import { requestAnimationFrame } from '../../../js/request-animation-frame';
+import {
+  add,
+  remove,
+} from '../../../js/class-list';
+import {
+  publish,
+  subscribe,
+} from '../../../js/pubsub';
+
+class AccordionItem extends UiEvents {
+  static DEFAULTS = {
+    container: '.js-accordion-item',
+    toggle: '.js-accordion-item__toggle',
+    body: '.js-accordion-item__body',
+    isOpen: 'is-accordion-item-open',
+  };
+
+  constructor(rootNode, options) {
+    // eslint-disable-next-line no-param-reassign
+    options = {
+      ...AccordionItem.DEFAULTS,
+      ...options,
+    };
+
+    super(rootNode, options);
```

Comment:
@TheDadi I renamed all `rootNode` to `wcNode` to clarify it's origin, please do so too.